docs/pages/access-controls/guides/passwordless.mdx

```diff
@@ -0,0 +1,354 @@
+---
+title: "Passwordless"
+description: Use passwordless authentication
+---
+
+Passwordless takes advantage of [WebAuthn](./webauthn.mdx) to provide
+passwordless and usernameless authentication for Teleport.
+
+## Prerequisites
+
+* A Teleport cluster with WebAuthn configured.
+  See the [Second Factor: WebAuthn](./webauthn.mdx) guide.
+* A hardware device with support for WebAuthn and resident keys, or a Mac with
+  biometrics / Touch ID.
+* A Web browser with [WebAuthn support](
+  https://developers.yubico.com/WebAuthn/WebAuthn_Browser_Support/).
+* A signed and notarized `tsh` for Touch ID.
+  [Download the macOS tsh installer](../../installation/#macos).
+
+## Step 1/3. Configure WebAuthn
+
+See the [Second Factor: WebAuthn](./webauthn.mdx) guide for an in-depth
+explanation of the WebAuthn configuration.
+
+A minimal configuration is shown below:
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+<Tabs>
+  <TabItem label="Static Config">
+    Auth Server `teleport.yaml` file:
+
+    ```yaml
+    # snippet from /etc/teleport.yaml:
+    auth_service:
+      authentication:
+        type: local
+        second_factor: on
+        webauthn:
+          rp_id: example.com
+    ```
+  </TabItem>
+  <TabItem label="Dynamic resources">
+    Create a `cap.yaml` file or get the existing configuration using
+    `tctl get cluster_auth_preference`:
+
+    ```yaml
+    kind: cluster_auth_preference
+    version: v2
+    metadata:
+      name: cluster-auth-preference
+    spec:
+      type: local
+      second_factor: on
+      webauthn:
+        rp_id: example.com
+    ```
+
+    Update the configuration:
+
+    ```code
+    $ tctl create -f cap.yaml
+    # cluster auth preference has been updated
+    ```
+  </TabItem>
+</Tabs>
+</ScopedBlock>
+
+<ScopedBlock scope={["cloud"]}>
+Create a `cap.yaml` file or get the existing configuration using
+`tctl get cluster_auth_preference`:
+
+```yaml
+kind: cluster_auth_preference
+version: v2
+metadata:
+  name: cluster-auth-preference
+spec:
+  type: local
+  second_factor: on
+  webauthn:
+    rp_id: example.com
+```
+
+Update the configuration:
+
+```code
+$ tctl create -f cap.yaml
+# cluster auth preference has been updated
+```
+</ScopedBlock>
+
+`rp_id` is the public domain of the Teleport Proxy Service, *excluding* protocol
+(`https://`) and port number.
+
+A Teleport cluster capable of WebAuthn is automatically capable of passwordless.
+
+## Step 2/3. Register
+
+Register your passwordless device using `tsh`:
+
+```code
+$ tsh mfa add
+# Choose device type [TOTP, WEBAUTHN, TOUCHID]: webauthn
+# Enter device name: bio
+# Allow passwordless logins [YES, NO]: yes
+# Tap any *registered* security key
+# Tap your *new* security key
+# MFA device "bio" added.
+```
+
+If you are using a hardware device, a passwordless registration will occupy a
+resident key slot. Regardless of your device type, passwordless registrations
```

Comment:
Readers might not know what a resident key slot means. Can we link to some resource here or word it in a way that help clarify?

Reply:
I'm struggling a bit to find a source that isn't developer focused, so I've added a brief explanation in text.

For reference, here are some the sources I think are best:

* https://developers.yubico.com/WebAuthn/WebAuthn_Developer_Guide/Resident_Keys.html (gets technical in about 2 paragraphs)
* https://www.w3.org/TR/webauthn/#discoverable-credential (definitely technical)